### ISSUE TITLE
Cell Delete Issue

### ISSUE THREAD

**Issue by CB750-Rider** [ragardner/tksheet#25]
Thanks for creating this repository. It was just what I needed for a project I was working on.

I ran into a problem when attempting to delete cells using the version on the master branch. The issue came from (_tksheet_main_table.py) at line 628. The original code was

`for r1, c1, r2, c2 in boxes:`

which had an issue because boxes was defined as [((10,0,11,1),"cells")] so the first list element only saw 2 values to unpack. I changed line 628 (and 649 because it looked like it would have the same problem) to

`for (r1, c1, r2, c2),_ in boxes:`

which seemed to fix the issue. 

I was able to recreate the problem with the following code:

```
import tkinter as tk
import tksheet

top = tk.Tk()

sheet = tksheet.Sheet(top)
sheet.pack()

sheet.set_sheet_data([[f"{ri} : {ci}" for ci in range(4)] for ri in range(10)])

sheet.enable_bindings(("single_select",
                       "row_select",
                       "column_width_resize",
                       "arrowkeys",
                       "right_click_popup_menu",
                       "rc_select",
                       "rc_insert_row",
                       "rc_delete_row",
                       "copy",
                       "cut",
                       "paste",
                       "delete",
                       "undo",
                       "edit_cell"))
top.mainloop()
```

Have a great day. Thanks for providing a great tool.

Lee

**Comment by ragardner** [comment]
@CB750-Rider Thank you very much for pointing this out, it should be fixed now in version `4.8.7` using your suggestion

**Comment by CB750-Rider** [comment]
You are welcome. I appreciate the useful module!
